benchmark/buffers/buffer-swap.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  method: ['swap16', 'swap32', 'htons', 'htonl'],
+  len: [4, 64, 512, 768, 1024, 1536, 2056, 4096, 8192],
+  n: [1e6]
+});
+
+// The htons and htonl methods below are used to benchmark the
+// performance difference between doing the byteswap in pure
+// javascript regardless of Buffer size as opposed to dropping
+// down to the native layer for larger Buffer sizes.
+
+Buffer.prototype.htons = function htons() {
```

Comment:
the term is used only in the benchmark and was selected because it's common for the operation. It's not actually used anywhere in the regular code or API.